### PR TITLE
fix: button with size and adds $

### PR DESCRIPTION
**Pull request by R-K-H**
Adds a dollar sign for total value, adjusts the max button so it doesn't overlap or have weird scale issues.

### CODE REVIEW

File: components/Markets/ConditionalMarketCard.tsx [hunk 1]
```diff
@@ -364,9 +364,11 @@ export function ConditionalMarketCard({ isPassMarket = false }: { isPassMarket?:
                 type="number"
                 value={amount || ''}
                 defaultValue={amount || ''}
+                rightSectionWidth={70}
                 rightSection={
                   <ActionIcon
-                    radius="md"
+                    w="80%"
+                    radius="lg"
```
Comment (swaggymarie) [inline components/Markets/ConditionalMarketCard.tsx]: I'd more put radius="sm" so it fits container

File: components/Markets/ConditionalMarketCard.tsx [hunk 2]
```diff
@@ -376,7 +378,7 @@ export function ConditionalMarketCard({ isPassMarket = false }: { isPassMarket?:
                   >
                     <Text size="xs">
                       Max{' '}
-                      {maxOrderAmount()
+                      {maxOrderAmount() && maxOrderAmount() < 1000
```
Comment (swaggymarie) [inline components/Markets/ConditionalMarketCard.tsx:381]: I don't understand why restricting maxOrderAmount to be under 1000, is it for formating ?

Comment (R-K-H) [reply components/Markets/ConditionalMarketCard.tsx:381]: Anything over 1000 overflows

File: components/Markets/ConditionalMarketCard.tsx (hunk 1 above)
Comment (R-K-H) [reply components/Markets/ConditionalMarketCard.tsx]: Will fix

**Comment by R-K-H** [comment]
Ready for re-review.

**Review by swaggymarie — APPROVED** [review]

**Pull request merged**